frontends/infinite-corridor/src/scss/combined.scss

```diff
@@ -1,28 +1,28 @@
 @use "material-components-web/dist/material-components-web";
-@use "ol-util/assets/variables.scss";
-
 @use "ol-util/assets/card";
 
-@use "button";
-@use "columns";
+@use "theme";
+@use "button" with ( $color: theme.$color-primary );
```

Comment:
@abeglova Btw re styles... `@use` seems great. It replaces `@import` in Sass's new module system... `@use <modulename> with ( <overrides> )` seems like it could be very useful for us.

Like, LearningResourceCard can specify its own styles, but we can override the variables it uses if we need to.